### PR TITLE
Fixes #18427: USEMETHODREPORTING in metadata documentation is missing

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/18427

### CODE REVIEW

**Comment by peckpeck** [comment]
Commit modified

File: src/reference/modules/reference/pages/techniques.adoc [hunk 1]
```diff
@@ -181,6 +182,20 @@ is equivalent to
 
 NOTE: `INPUT` fields are automatically escaped, meaning any quote will be written in the policies as \" ; and any backslash will be written as \\
 
+=== Report keys in the <SECTION> tags
+
+The REPORTKEYS element help detecting the proper reporting key based when it cannot be know in advance. The key can ocntant variables that will be matched based on a regex.
```
Comment (amousset) [inline src/reference/modules/reference/pages/techniques.adoc:187]:
```suggestion
The REPORTKEYS element helps detecting the proper reporting key when it cannot be known in advance. The key can contain variables that will be matched based on a regex.
```

Comment (peckpeck) [reply src/reference/modules/reference/pages/techniques.adoc:187]: erf

**Comment by peckpeck** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**